src/shared/ui/tooltip/tooltip.stories.tsx

```
@@ -1,330 +1,317 @@
-                        </ul>
-                    </div>
-                </Tooltip.Content>
-            </Tooltip>
-        </div>
-    ),
+  args: {
+    children: null,
+  },
+  render: () => (
+    <div className="flex items-center gap-2">
+      <label className="text-sm font-medium">상품 정보</label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)